### ISSUE TITLE
Improve paper + incorporate comments

### ISSUE THREAD

**Issue by santiagohermo** [diegogentilepassaro/min_wage_rent#65]
We have received several comments on the current version of the paper. In this issue, I will tackle some of those comments. Broadly, 

- I'll work on rewriting and make some other changes to improve clarity
- I'll answer to the point-by-point comments by Jesse: [00main.pdf](https://github.com/diegogentilepassaro/min_wage_rent/files/5461611/00main.pdf)

_EDIT Nov 26, 2020_: We will update the draft fully in this branch as well.

**Comment by santiagohermo** [comment]
I started to address some of the issues here in branch `issue57`. After #63 is merged back to master, I will open a new branch

**Comment by santiagohermo** [comment]
Hey @santiagohermo @gabrieleborg! I did a few things we talked about last week.

- I spent some time thinking of our empirical model, in particular in the Arellano Bond magic. I started [this note](https://github.com/diegogentilepassaro/min_wage_rent/blob/3826ee0b65fdd46415ced1e145f782d4149a3ea6/analysis/notes/empirical-strategy.pdf) to clearly write down the assumptions we make in each model.
- I did a lot of re-writing of the introduction, following Jesse's comments and my thoughts from the point above.
   - Paper with updated abstract and introduction: [00main.pdf](https://github.com/diegogentilepassaro/min_wage_rent/files/5470013/00main.pdf)
   - Version with answer to Jesse's comments in the introduction: [00main_JS_SH.pdf](https://github.com/diegogentilepassaro/min_wage_rent/files/5470017/00main_JS_SH.pdf)

Could you take a quick look at the updated introduction? I plan on sending this before tomorrow at noon. Thanks!

During the week I'll continue to address the comments we received and improving the draft.

**Comment by diegogentilepassaro** [comment]
@santiagohermo please find my comments below:

- Comments on the empirical strategy notes [here](https://github.com/diegogentilepassaro/min_wage_rent/files/5470122/empirical-strategy_DGP.pdf).

- Comments on the updated version [here](https://github.com/diegogentilepassaro/min_wage_rent/files/5470123/00main_DGP.pdf). The only other comments about this is that the last paragraph of page 3 in the old version (now removed) used to convey the message that using month-to-month variation is much cleaner than using year-to-year. Can we add something like that to the new version?

**Comment by santiagohermo** [comment]
Thanks @diegogentilepassaro! I implemented most of your changes. On your comments:

- Thanks for the comments on the empirical strategy notes! I changed a few things based on them, and here are some replies: 
[empirical-strategy_DGP_SH.pdf](https://github.com/diegogentilepassaro/min_wage_rent/files/5471040/empirical-strategy_DGP_SH.pdf). I thought this as an informal note for us to better understand what we are doing, so feel free to implement changes here.

- Thanks for flagging this! I removed one part there that Jesse said he didn't follow. In particular, before we said to things there:
   1. When you use year-to-year some units are "partially treated" because they change their MW in some middle month.
   2. Some jurisdictions have MW changes on many subsequent years, making it hard to disentangle which MW the effect was coming from.
 
  Point 1 is still in the paper. Jesse said he didn't follow point 2, which is way I removed it. I think that intuition was coming from the event-studies that use relative time dummies, not sure if they still apply (we should try our fd model with bigger windows to check). I'll leave it as it for now, and we can discuss it later and in any case we can easily bring it back later.

I'm sending the email now. Thanks!

**Comment by gabrieleborg** [comment]
@santiagohermo 

I added my comments to the doc with Diego's on it:[00main_DGP_GB.pdf](https://github.com/diegogentilepassaro/min_wage_rent/files/5471257/00main_DGP_GB.pdf). Most of them are in the intro but are minor english corrections. I will now add the missing references to rent data!

**Comment by santiagohermo** [comment]
Thanks @gabrieleborg! I implemented most of your changes as well. In order to track what stuff I implemented, I also replied to your and @diegogentilepassaro's comments here: [00main_DGP_GB_SH.pdf](https://github.com/diegogentilepassaro/min_wage_rent/files/5471667/00main_DGP_GB_SH.pdf)

Regarding the style for enumeration. I thought about this at some point, and Google revealed that "First, ... Secondly, .. Thirdly ..." is common usage. For example, see [this grammar stackexchange post](https://english.stackexchange.com/questions/174847/first-second-third-and-finally). "First, ... Second, ..." and "Firstly, ... Secondly, ..." are also ok, although "firstly" sounds weird. I guess it's a matter of style, so we can use either. If you feel strongly about "First, ..., Second, ..." feel free to implement it!

**Comment by gabrieleborg** [comment]
That's fine then @santiagohermo ! I thought you couldn't say that but then I am ok with it 😊

**Comment by santiagohermo** [comment]
Broad structure of results (please check @gabrieleborg): 
- Main table and two-panel figure from #67 
- Identification threats and alternative models
- Composition threats (add placebo with number of listings)
- Heterogeneity
- Experienced MW
- Benchmarking / discussion

**Comment by gabrieleborg** [comment]
Thanks @santiagohermo ! 

I think we ended up agreeing in leaving heterogeneity after experienced MW. So that would be: 

- Main table and two-panel figure from #67 
- Identification threats and alternative models
- Composition threats (add placebo with number of listings)
- Experienced MW
- Benchmarking (discussion?)
- Heterogeneity
- discussion ?

I think we can figure out where to add a proper discussion in next week meeting after some updates

**Comment by gabrieleborg** [comment]
As per [my post](https://github.com/diegogentilepassaro/min_wage_rent/pull/71#issuecomment-732430754) in #71, I will start updating the various part of the analysis to match the structure Santi introduced in the main `analysis/first_difference`. I will update a bit the outputted tables and figures to better follow the narrative of our results (see post above).

**Comment by santiagohermo** [comment]
As I waited for the file for #62 I made a few changes to the structure of the draft (importantly, I dropped the numbering of the files of the sections as this is likely to change) and I rewrote some bits of the data section. @gabrieleborg

**Comment by gabrieleborg** [comment]
@santiagohermo @diegogentilepassaro quick update: I have started improving the draft. At the moment I have:

1. Re-viewed the data section. I added a picture showing the variation in Mw we are using (two histograms, one for the density of d_ln_mw and one for the distribution of periods when the changes happen)
2. Update the methodological section reflecting the fact that we now use a model with Econ control as our baseline. Here there are a couple of issue still to tackle: 
    - At the moment I don't go into details on which sectors are we using as controls. Shall we describe these here already, or shall we wait in the result section? 
    - I wasn't sure where and how to frame the Jesse's comment on the impossibility of disentagnling the presence of: a) confounder effects showing up in the leads' coefficients; b) anticipatory effects also showing up in the leads coefficients (can you confirm this was the main point he made?) I think we should do that when we introduce the dynamic model. I also think we should assume the lack of confounders and say we are using leads to test for anticipatory effect. This is because we could technically check for the former (at least partially) by including Econ control leads (don't know if we have time, but it shouldn't be long anyway).

**Comment by santiagohermo** [comment]
I just committed several updates to the empirical strategy section @gabrieleborg @diegogentilepassaro. I adressed the points you raised above Gabri. Point by point, I did the following:

- Left a few questions and placeholders for things to think through in the code
- Fixed equations by indexing everything by $c$ and I rephrased important chunks of code
- Added a mention of the sectors we use as controls
- Added some argument relating Jesse's point about confounding economic shocks with anticipatory effects 
- Wrote some detailed notes for a few tables

Please review and make sure it's correct!

**Comment by santiagohermo** [comment]
(I'll later review your other changes @gabrieleborg. Thanks!)

**Comment by santiagohermo** [comment]
One question about the structure of the paper that came up as I was revising the empirical strategy section yesterday @diegogentilepassaro @gabrieleborg. Should we describe the experienced MW there? If we think that it aids in our identification, then probably we should.

The structure (titles) could be some like
- Data and Sample Selection Criteria
- Empirical Strategy and Identification (discussing the experienced MW)
- The elasticity of rents to the MW: baseline results (results of static and dynamic models + heterogeneity)
- Tracking low-wage workers: the experienced minimum wage (results using experienced MW)
- Discussion (benchmarking and stuff)
- Conclusion

**Comment by diegogentilepassaro** [comment]
@santiagohermo I think that sounds like a good idea. At least introducing the experience MW and stating how it helps to identification at the empirical strategy section is definitely a good thing.

**Comment by gabrieleborg** [comment]
Morning @santiagohermo @diegogentilepassaro 

introducing the experienced MW in the Empirical Strategy sounds find to me! I have already started adding that section in results, so  you may want to have a look there to move some paragraphs up in the empirical strategy :) 

Also, yesterday I have: 

- updated the introduction paragraph in results, where we summarized our findings
- revisited the baseline results section, updating tables, figures and comments around it
- added the placebo exercise with number of listings
- updated the external validity section with new tables and figures 

I have also started working on including the Experienced MW section. As per @santiagohermo's comment [here](https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-735784394), feel free to re-struture this part, which I haven't finished yet. 

Today I am planning to: 
- finish the results/experienced MW section
- add the benchmark section 
- (if I have time) start reviewing a bit discussion/conclusions

**Comment by santiagohermo** [comment]
Hey fellow coauthors @gabrieleborg @diegogentilepassaro. I completed a review of the Data Section. I did the following:

- Changed conceptual approach. Now we have 4 sections, in the first two we describe the Zillow data and the MW (including construction of experienced MW), in the third we describe other data sources, and in the fourth we describe our panel and the sample used for estimation.
- I fixed the descriptive tables (thanks for running that Gabri!). We have table 1, which compares different samples of zipcodes in a few demographics, table 2 which shows descriptive statistics of our estimating sample for key variables, and table A.1 which is a longer version of table 2.
- I added [`analysis/sumstats`](https://github.com/diegogentilepassaro/min_wage_rent/tree/2292a2b15c23720876f1d65a631d78d2ab43c2fc/analysis/sumstats), which at the moment simply counts MW events (see lof files in `output`)

I left a couple of questions and comments for things to add in the [.tex file](https://github.com/diegogentilepassaro/min_wage_rent/blob/2292a2b15c23720876f1d65a631d78d2ab43c2fc/paper_slides/paper/data_sample.tex). Please review. Importantly, it would be useful if @gabrieleborg makes a nicer explanation of the construction of weights using LODES for the experienced MW.

Please review and make improvements!

**Comment by gabrieleborg** [comment]
Thanks @santiagohermo @diegogentilepassaro ! 

I will address your comments tomorrow morning :) 

On my side, I have: 

- Finished updating the baseline results for heterogeneity. Here there is still a couple of reference missing but I gave it a first pass and updated the table. When adding the controls, most of the expected results remain, but the one with median income mostly disappear. I provided an explanation on why that could be, and added a column to support my explanation (basically, median income might still be associated with variable shares on Mw workers, depending on the bite of MW, i.e. the Mw/median income ratio in a zip code). 
- added the experienced MW result section. Here I provide an explanation of why it is important to track that, and show results. For the moment I actually **also added** two columns with the indicators we discussed this morning. If we decide not to keep them, we can easily remove them. For this reason, the comment on the final table is a bit incomplete. 
- added the whole benchmarking exercise under the _discussion_ section. I don't know if we want to keep it here, or if you guys want to add a more broader discussion also in the light of the benchmark findings. 

Tomorrow I will address Santi's comments and start tying the loose ends (notes, etc). I will also review the conclusion to integrate the new stuff we added. Please let me know if you can read the stuff I added, especially where it still needs some additional expansion!

**Comment by diegogentilepassaro** [comment]
@gabrieleborg and @santiagohermo, please find [here](https://github.com/diegogentilepassaro/min_wage_rent/files/5629777/Main_comments_intro_DGP.pdf) some comments on the abstract and introduction. Given that it is not yet updated, it should be thought of as a document to start the discussion on that update. Feel free to add more comments and after we all review the whole thing we can quickly discuss and implement what we chose as the last step (along with changing the conclusion) before submitting. 

Will move on to data now! Thanks for the amazing work 👍

**Comment by diegogentilepassaro** [comment]
@gabrieleborg and @santiagohermo, please find [here](https://github.com/diegogentilepassaro/min_wage_rent/files/5636947/Main_comments_data_DGP.pdf) my comments on the data section. Given that we are 3 people touching the draft it seems that collecting comments in PDF and then applying only the ones that we all agree is a good way to stage and organize the production of the coming draft! 

Let's discuss this in today's meeting.

**Comment by santiagohermo** [comment]
Collecting comments in a pdf sounds like a great idea @diegogentilepassaro! Could you include comments about the empirical strategy section in the pdf in [this comment](https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-738065550), together with the data section? That way they'll be easier to address.

**Comment by diegogentilepassaro** [comment]
@santiagohermo I would love to do that, but unfortunately, that would be weird because the PDFs are different due to the fact that some updates happened between the time I posted both comments. I propose that we review all sequentially by section just in case. BUt we can discuss this in our meeting!

**Comment by santiagohermo** [comment]
One small thing @gabrieleborg. The current main figure of the dynamic model doesn't use the controls. I will say in the text that it does, can you include them after your exploration? Thanks!

**Comment by gabrieleborg** [comment]
Absolutely @santiagohermo

**Comment by diegogentilepassaro** [comment]
@gabrieleborg just in case it inspires you or helps you with the controls, take a look at section 4.1.2 in (Meer and West](https://www.nber.org/system/files/working_papers/w19262/w19262.pdf)!

**Comment by diegogentilepassaro** [comment]
More ideas in page 10 [here](http://ftp.iza.org/dp3499.pdf)!

**Comment by gabrieleborg** [comment]
thanks @diegogentilepassaro !

**Comment by diegogentilepassaro** [comment]
Please find my comments on the static and dynamic models parts of the empirical strategy [here](https://github.com/diegogentilepassaro/min_wage_rent/files/5640162/Main_comments_static_dynamic_DGP.pdf)

**Comment by santiagohermo** [comment]
In https://github.com/diegogentilepassaro/min_wage_rent/commit/c788c9dd6d8981460a689315a0bebf353c896122 I applied a bunch of updates to the data section following your comments @diegogentilepassaro. I implemented most of the stuff (although not all of them). See answers to your comments in [Main_comments_data_DGP_SH.pdf](https://github.com/diegogentilepassaro/min_wage_rent/files/5644219/Main_comments_data_DGP_SH.pdf)

fyi @gabrieleborg

**Comment by diegogentilepassaro** [comment]
Thanks for the work @santiagohermo! Most of your decisions look great to me. The very few that don't or that are worth discussing I added my answers [here](https://github.com/diegogentilepassaro/min_wage_rent/files/5647227/Main_comments_data_DGP_SH_DGP.pdf).

Happy to discuss soon :D

**Comment by santiagohermo** [comment]
I wrapped up a review of the results section, **first two subsections only** @diegogentilepassaro @gabrieleborg . Because I'm updating tables and figures, this is taking more time than expected. 

I have to work many hours for my RAship this next couple of days, so I was wondering if you guys  could help our here. A short list of some things to do:

1. Review the open PR #72 and review the discussion section, with a focus on a nice explanation of the benchmarking
2. Add notes to figures and tables in appendix, following style of notes in main body of paper
3. Review heterogeneity subsection in paper. Include there results using LODES workplace and residence location
4. Review experienced mw section and, in particular, add table with results using both measures together

If any of you guys can start with either points 1 or 2 that'd be great. My plan is to complete 3 this weekend, and start with 4 hopefully Sunday night/monday.

**Comment by diegogentilepassaro** [comment]
That sounds like a great plan @santiagohermo! I am happy to review PR#72 and the discussion section! WIll move on to do that soon 👍  Once @gabrieleborg and I wrap that up I can also take care of 2.!

Let's finish this push brodas 💘

**Comment by gabrieleborg** [comment]
thanks @santiagohermo  and @diegogentilepassaro ! I I will start with 2 on Santi's list and help Diego in the benchmarking review. I will also try to start thinking some broader conclusions to draw from the overall results in the paper, so to formulate a clear and concise discussion :)

**Comment by gabrieleborg** [comment]
@diegogentilepassaro @santiagohermo fyi I added a discussion subsection after the benchmarking exercise. That is a first pass in drawing some general conclusions about the significance of our results. We may decide to move it either in the conclusions, or wherever.

**Comment by santiagohermo** [comment]
Thanks @gabrieleborg! That's a very useful section. We'll progressively improve it this week.

I completed a pass on results and started with experienced MW. [Here is the current paper](https://github.com/diegogentilepassaro/min_wage_rent/blob/7217c109ff3d0f759e9594abf53dd52c96f40cd7/paper_slides/paper/Main.pdf). A few things I noticed and wanted to briefly discuss with you @diegogentilepassaro @gabrieleborg  

- On heterogeneity. 
   - I changed the table and included 3 socio-demographics and 2 variables from LODES. I think this transmits the message that what matters is where MW workers live. What do you guys think? I'm ok with this, but maybe you guys want to use more variables? We could easily make another table for the appendix.
   - I created Figure B.7 which shows the heterogeneity results with LODES data using both the state share and zipcode share definitions. I framed it as a robustness exercise in the text.

- On experienced MW. 
   - I'll work on improving the table today. My plan is to go ahead with using both variables in the regression. Any comments//ideas? Maybe you can review later and we add/remove stuff based on comments?
   - I think we can potentially include some more maps using the rents variable. Let's discuss this tomorrow!

- On style of the paper.
    - I noticed with use inconsistent styles in titles of subsections, tables and figures (sometimes we write like "The Effect of Minimum Wage", and other "The effect of minimum wage"). I think it makes sense to do whatever the AER does here. Just need to check a few random papers, and apply the style we want throughout the paper.
    - Do you think we can change the `autoref` for appendix figures and tables to print "Appendix Figure XX", or "Appendix Table YY". Right now I'm writing "Figure XX in the appendix"/"Table YY in the appendix", which is not optimal.

**Comment by diegogentilepassaro** [comment]
Thanks, @santiagohermo! I am reviewing the writing of sections 3.3, 3.4 and 4. now. I will post a round of comments very soon. 

As for heterogeneity your choices sound good!

On experienced MW, going ahead with both variables is a great idea and it is consistent t one of the paragraphs in the draft that argues in favor of using the experienced MW. Once we have the table post it and I can review immediately!

On style, harmonizing things is a good idea! AER puts titles as "The Effect of Minimum Wage". As for the workings of \autoref I have no idea. Some googling gave the following two, but they are not exactly what we are looking for, so maybe we have to do it manually:

- https://tex.stackexchange.com/questions/130096/autoref-to-appendix-with-appendix-package
-https://stackoverflow.com/questions/3611426/latex-trying-to-use-ref-or-autoref-to-a-label-in-an-appendix-shows-up-emp/3617238

**Comment by santiagohermo** [comment]
Thanks @diegogentilepassaro! I'll update here once I have the table for the exp MW

On harmonizing. I checked this [AER article](https://pubs.aeaweb.org/doi/pdfplus/10.1257/aer.101.7.2980) and it seems you are write. I'll move to that title convention then.

On the `autoref` command. Maybe @gabrieleborg understand how to fix it? (I remember he fixed pretty easily the one for equation). If not, I'll read those entries and figure it out.

**Comment by gabrieleborg** [comment]
Thanks @santiagohermo ! that sounds fine to me. 

On heterogeneity: 

- I saw you took out unemployment and teen share. I think we should put at least one of those two demos back. If the goal is to show that the effect is localized mostly in places where MW workers are likely to live, younger age and unemployment rate are better predictors than race. I think it's ok to keep race as it is always an important topic, but maybe let's add unemployment? This would also help in explaining why we don't see much of an impact in median income (another variable could be the share of people living in poverty, that we have). 

- On experienced MW: 
    - I am fine with you experimenting :). 
    - Why do you want to add rent maps in this section? we already have some but they weren't very informative..

- On style: I think we should capitalize words in titles (sections, graphs, tables, etc). This is a pretty standard style in all major publications! As for autoref, I think we can tweak it accordingly.  

I have reviewed the data section, adding few comments and addressing some of the remaining few. you can find them here:
[Main_comments_data_DGP_SH_DGP_GB.pdf](https://github.com/diegogentilepassaro/min_wage_rent/files/5655026/Main_comments_data_DGP_SH_DGP_GB.pdf)

If you guys are still working on the draft, I can update the benchamrking table in #72.

**Comment by santiagohermo** [comment]
Thanks @gabrieleborg! Your initial comments make sense to me.

I'll review your comments and edits to the data section soon, to make sure I'm up-to-date with the decisions we are making there. Great that you could take a look and make sure that our description of the data is broadly accurate!

**Comment by gabrieleborg** [comment]
Quick update on the "Appendix" label. I tried for a while to fix it but it is taking more time it should ahah. So, for the moment I simply manually added the word Appendix in front of the \autoref command, so we get for example "Appendix Table A.1". It's not ideal but it was the solution requiring the least amount of changes.

**Comment by diegogentilepassaro** [comment]
@santiagohermo and @gabrieleborg my comments on sections 3.3, 3.4, 4 and 4.1 are 
[here](https://github.com/diegogentilepassaro/min_wage_rent/files/5655580/Main_sections3_3_to_4_1_DGP.pdf). Sorry that there are quite some things to discuss!

**Comment by santiagohermo** [comment]
### Comment to track annotated PDFs and reviews
fyi @diegogentilepassaro @gabrieleborg . Feel free to edit once you comment some new stuff

| Section                       | DGP | SH |  GB | latest file  |
| ---                               | ---    |  --- | ---  | --- |
| Intro                            |    Y    |  N  |   N  |  [Main_comments_intro_DGP.pdf](https://github.com/diegogentilepassaro/min_wage_rent/files/5660291/Main_comments_intro_DGP.pdf) |
| Data                            |   YY   |  Y   |   Y   | [Main_comments_data_DGP_SH_DGP_GB.pdf](https://github.com/diegogentilepassaro/min_wage_rent/files/5660282/Main_comments_data_DGP_SH_DGP_GB.pdf) |
| Emp Str I                     |    Y    |  N  |  N  |  [Main_comments_static_dynamic_DGP.pdf](https://github.com/diegogentilepassaro/min_wage_rent/files/5660290/Main_comments_static_dynamic_DGP.pdf) |
| Emp Str II and Results  |  Y   |  N |  N  | [Main_sections3_3_to_4_1_DGP.pdf](https://github.com/diegogentilepassaro/min_wage_rent/files/5660299/Main_sections3_3_to_4_1_DGP.pdf) |
| Exp MW                       |
| Discussion                   |
| Conclusion                   |

**Comment by diegogentilepassaro** [comment]
Thanks for creating that table @santiagohermo! Very organized :D I will repost it with the addition of my comments for sections 4.2 and 4.3.

### Comment to track annotated PDFs and reviews
fyi @diegogentilepassaro @gabrieleborg . Feel free to edit once you comment some new stuff

| Section                       | DGP | SH |  GB | latest file  |
| ---                               | ---    |  --- | ---  | --- |
| Intro                            |    Y    |  N  |   N  |  [Main_comments_intro_DGP.pdf](https://github.com/diegogentilepassaro/min_wage_rent/files/5660291/Main_comments_intro_DGP.pdf) |
| Data                            |   YY   |  Y   |   Y   | [Main_comments_data_DGP_SH_DGP_GB.pdf](https://github.com/diegogentilepassaro/min_wage_rent/files/5660282/Main_comments_data_DGP_SH_DGP_GB.pdf) |
| Emp Str I                     |    Y    |  N  |  N  |  [Main_comments_static_dynamic_DGP.pdf](https://github.com/diegogentilepassaro/min_wage_rent/files/5660290/Main_comments_static_dynamic_DGP.pdf) |
| Emp Str II and Results I  |  Y   |  N |  Y  |[Main_sections3_3_to_4_1_DGP_GB.pdf](https://github.com/diegogentilepassaro/min_wage_rent/files/5660483/Main_sections3_3_to_4_1_DGP_GB.pdf) |
| Results II  |  Y   |  N |  N  |[Main_4_2_and_4_3_DGP.pdf](https://github.com/diegogentilepassaro/min_wage_rent/files/5660548/Main_4_2_and_4_3_DGP.pdf) |
| Exp MW                       |
| Discussion                   |
| Conclusion                   |

**Comment by santiagohermo** [comment]
Thanks @diegogentilepassaro! I was going crazy scrolling up to find old annotated files haha, so the table came up

I suggest we keep track of everything in the same comment. I will therefore edit your comment, and I suggest @gabrieleborg does the same in the future.

I completed a new pass on the comments to the data section and applied a few more fixes. I thus updated your comment with the new file. One important thing I noticed is that the variables `dactual_MW` is wrong in the panel. When you run either
> list if dactual_mw < 0

or 
> list if D.actual_mw < 0

in the main fd panel we get different results. We should definitely check this!

**Comment by gabrieleborg** [comment]
@santiagohermo  I checked those differences. There are 3 observations (zipcode-month) in Miami MSA that report a decrease in MW (you can check how the story evolved [here](https://www.floridaphoenix.com/blog/what-happened-to-raising-the-minimum-wage-new-fl-supreme-court-wont-take-up-the-miami-beach-wage-case/) and [here](https://www.wageandhourlawupdate.com/2019/02/articles/minimum-wage/miami-minimum-wage-ordinance-remains-invalid-after-review-denied-by-florida-supreme-court/)). dactual_mw don't track those changes because we actively dismissed those: you can see it in [`derived/zipcode_month/code/clean_gen_label.do`](https://github.com/diegogentilepassaro/min_wage_rent/blob/master/derived/zipcode_month/code/clean_gen_label.do) line 36: https://github.com/diegogentilepassaro/min_wage_rent/blob/12e2144a7dafa50dc5b967f4f5a616a7e2e67045/derived/zipcode_month/code/clean_gen_label.do#L36

**Comment by santiagohermo** [comment]
Good to know @gabrieleborg! So I guess that, since we use the `D.` operator throughout, we are using negative changes in the regression right?

**Comment by gabrieleborg** [comment]
@santiagohermo well not everywhere. In the heterogeneity analysis, and in general whenever we have to interact D.ln_mw with something else, we have to switch to d_ln_mw because state does not handle time series operators and interactions together. So in this sense, those effect may be slightly affect that section of the results, even though being just 3 observation I doubt they actually change our estimates.

**Comment by santiagohermo** [comment]
I see! I agree that it probably won't matter, but just to be consistent we should probably delete that line and re-run everything. :sleepy:

**Comment by diegogentilepassaro** [comment]
Just updated [this commen](https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-740755114) with my comments for EXP MW section.

**Comment by gabrieleborg** [comment]
Updated [task table](https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-740755114)

**Comment by santiagohermo** [comment]
I added the table with changing window and rephrased a bit Gabri's write-up (we have to introduce this issue before mentioning the point estimate of the appendix table I think). I flag that we still need to rephrase a bit the assumptions required for the interpretation of the pre-trends

fyi @diegogentilepassaro @gabrieleborg

**Comment by gabrieleborg** [comment]
Hey @santiagohermo @diegogentilepassaro ! I finished reviewing my sections. Please see my notes in the PDFs for comments and clarifications. I will proceed and start reviewing, and updating, the introduction !

**Comment by santiagohermo** [comment]
Merged PR #72 to #65 in [this commit](https://github.com/diegogentilepassaro/min_wage_rent/commit/2e4a445086cfb2e0f279ad303fca6b278e0b8af4)

**Comment by gabrieleborg** [comment]
Updated Introduction. See [pdf in comment table](https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-740755114)

**Comment by diegogentilepassaro** [comment]
@gabrieleborg and @santiagohermo I have updated the empirical strategy section and therefore the table [here](https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-740755114). I made many changes but some important notes:
- My tex does not compile the draft still. With the staged output it was about the eps files. With the whole draft it compiles about the bab package. I have committed without compiling the pdf. 
- Per discussion in #73 I have changed in both empirical strategy and data the start date of our panel to February 2010. 
- I added a phrase about autocorrelation in the errors of the level model. We need to decide where in the Appendix to place the corresponding table. What do you guys think?
- I updated the assumptions. I rephrased the static one and for the dynamic model it is needed strict exogeneity as well, so it is just the same. We can discuss why in the meeting.
- In the future we should consider adding yearly zipcode population from the ACS, or monthly unemployment measures at the county level from LAUS (https://www.bls.gov/lau/) as controls, as proxies for demand to make our economic controls even more credible.
- In footnote 20 in the PDF we needed some argument on why the economic controls are unaffected. A reference of the document with the share of MW by industry needs to be added and once #75 is through we need to add the figure numbers because I refer to them as well. 
- There is a sentence that I don't think is correct (I flagged before but we accidentally skipped in our last meeting) and must be checked and discussed. It comes after equation 3 and it reads: "However, increasing s implies dropping observations at the start and the end of the panel, which decreases our sample size and makes the results less reliable." This should be the only unchecked thing in the updated pdf comments that lives in the progress table.

**Comment by santiagohermo** [comment]
Thanks @diegogentilepassaro! I'm still working on completing a run of the placebo issue (related to that, I fixed a minor thing in the addition of tot_emp_wage_countymonth.dta @gabrieleborg. I switched to using naics codes before, so your previous code wasn't working). I will be done later today 

Some comments on your edits: 

> * Per discussion in #73 I have changed in both empirical strategy and data the start date of our panel to February 2010.

Should we also update the descriptive tables? They start in January 2010

> * I added a phrase about autocorrelation in the errors of the level model. We need to decide where in the Appendix to place the corresponding table. What do you guys think?

This is a marginal analysis, so I think the appendix is fine. But I'm happy with whatever @gabrieleborg decides here

> * I updated the assumptions. I rephrased the static one and for the dynamic model it is needed strict exogeneity as well, so it is just the same. We can discuss why in the meeting.

I don't understand how this can be. I think we know the assumption of strict exogeneity doesn't hold for the dynamic model, since current minimum wage changes are correlated with future rents (next period). How can then the assumption be the same?

Alternatively, we can make `la del 30`. Simply state the assumption for the static model in the text, and then don't say anything for the dynamic one.

> * In the future we should consider adding yearly zipcode population from the ACS, or monthly unemployment measures at the county level from LAUS (https://www.bls.gov/lau/) as controls, as proxies for demand to make our economic controls even more credible.

Sounds good!

> * In footnote 20 in the PDF we needed some argument on why the economic controls are unaffected. A reference of the document with the share of MW by industry needs to be added and once #75 is through we need to add the figure numbers because I refer to them as well.

I'll be done with #75 soon! I think that referencing the appendix (instead of the footnote) should be ok there

> * There is a sentence that I don't think is correct (I flagged before but we accidentally skipped in our last meeting) and must be checked and discussed. It comes after equation 3 and it reads: "However, increasing s implies dropping observations at the start and the end of the panel, which decreases our sample size and makes the results less reliable." This should be the only unchecked thing in the updated pdf comments that lives in the progress table.

Say you are estimating the static model. For dec 2019, your variables are constructed as the difference between nov and dec 2019, so nothing is missing and the model works. Now, say you add one lead and one lag. When t = dec 2019 you need jan 2020 to compute the lead, which is not available. As a result, dec 2019 is not used in the model with 1 lead and lag (since one covariate is missing, my guess is that STATA drops that observation). 

Following that reasoning, if we use 9 leads and lags then we are using 1 and 1/2 years fewer data to run our models. That's what I tried to convey in that sentence. Do you agree?

---
A few further comments:

- I noticed that you added two unwanted files from the tex compilation. I deleted them [here](https://github.com/diegogentilepassaro/min_wage_rent/commit/a610dd79eeebc66274a7fe82b71e73f3e17e8c8f#diff-e7460fc9d63952bb4703d999c6eee1c6c7c9aacadac026d49aaa18d93dbb577b). Try not to commit them later if they show up (they are added to gitignore, expect the `paper_slides/paper/Main.synctex(busy)` one).
- Note how [in the commit](https://github.com/diegogentilepassaro/min_wage_rent/commit/a610dd79eeebc66274a7fe82b71e73f3e17e8c8f) the lines you added are too long. We've been manually wrapping lines with @gabrieleborg to try to improve visualization in github. If you can stick to this convention that'd be great!
- On the added text. I noticed that you added a footnote after the "sequential exogeneity" part (L192/172 in [the diff](https://github.com/diegogentilepassaro/min_wage_rent/commit/a610dd79eeebc66274a7fe82b71e73f3e17e8c8f#diff-e7460fc9d63952bb4703d999c6eee1c6c7c9aacadac026d49aaa18d93dbb577b)). Per convention we are placing all footnotes after a punctuation mark, so that one should live after the dot in that sentence.

I'll make a deeper pass on the content later. Thanks again!

**Comment by gabrieleborg** [comment]
Thanks @diegogentilepassaro ! 

> * My tex does not compile the draft still. With the staged output it was about the eps files. With the whole draft it compiles about the bab package. I have committed without compiling the pdf.

Did you try the trick I wrote in Whatsapp? 

> * Per discussion in #73 I have changed in both empirical strategy and data the start date of our panel to February 2010.

Good to me. Let's remember to update the summary stat table. 

> * I added a phrase about autocorrelation in the errors of the level model. We need to decide where in the Appendix to place the corresponding table. What do you guys think?

I am fine with add the table in Appendix! I guess we should just add that in the correct position in terms of when it's mentioned in the text. 

> * I updated the assumptions. I rephrased the static one and for the dynamic model it is needed strict exogeneity as well, so it is just the same. We can discuss why in the meeting.

I also reviewed that and took some notes. Let's tackle this in the call!

> * In the future we should consider adding yearly zipcode population from the ACS, or monthly unemployment measures at the county level from LAUS (https://www.bls.gov/lau/) as controls, as proxies for demand to make our economic controls even more credible.

100%

> * In footnote 20 in the PDF we needed some argument on why the economic controls are unaffected. A reference of the document with the share of MW by industry needs to be added and once #75 is through we need to add the figure numbers because I refer to them as well.
I
 think I already added a reference to the BLS report on Minimum wage workers. In addition, we should mention the appendix exercise Santi is checking now !

> * There is a sentence that I don't think is correct (I flagged before but we accidentally skipped in our last meeting) and must be checked and discussed. It comes after equation 3 and it reads: "However, increasing s implies dropping observations at the start and the end of the panel, which decreases our sample size and makes the results less reliable." This should be the only unchecked thing in the updated pdf comments that lives in the progress table.

Let' discuss that

**Comment by santiagohermo** [comment]
Just fyi @diegogentilepassaro @gabrieleborg. I re-ran our main analysis allowing for negative MW changes. Results are nearly identical. We should check final numbers in the paper in a final past though

One minor thing is that I seem to have a few more observations than the committed run (no idea why). See you in the meeting!

**Comment by santiagohermo** [comment]
Here is a summary of our discussion about the models @gabrieleborg @diegogentilepassaro 

There are two potential threats for our strategy:
1. Anticipatory effects from the housing market (before "fixed supply", incorrectly). This would make the increase in rents happen earlier, and thus our coefficient would be wrong
2. Unobserved shocks causing both rents and MW changes.

The third threat is feedback, but we assume it away with the strict exogeneity assumption already.

### Two stories to interpret no-pretrends after conditioning for economic shocks

- The economic shocks capture all variation arising from (2), i.e., unobserved shocks causing both rents and MW. Assume that this is the case. Then, interpret the absence of pre-trends as no anticipatory effects, which kills (1). Provide further evidence on no anticipatory effects with the number of listings regression.

- Assume no anticipatory effects (1), and provide evidence in favor of this assumption with the listings regression. Interpret absence of pre-trends as evidence of no further economic shocks causing both, which kills (2). 

I'm happy with any story that you guys prefer! We just need to be consistent throughout the paper.

**Comment by gabrieleborg** [comment]
**Final Tasks!**

- Gabri
    1. Update Discussion Section (Benchmark Table + comments + policy implications) [0%]
    2. PR Diego's table on entry [0%]

**Comment by santiagohermo** [comment]
Hey guys @diegogentilepassaro @gabrieleborg. I ran some simulation to understand the strict exogeneity assumption. [This is the pdf](https://github.com/diegogentilepassaro/min_wage_rent/blob/96572802b9eb0bdcf28610af4904ac18539ab34a/analysis/notes/exogeneity_assumptions.pdf)

I simulated models with and without 2-period dynamic effect, and with and without violations of strict exogeneity. I think the answer is that both Diego and I are right. What we see is consistent with strict exogeneity, and with a violation of it. Let me know what you guys think!

**Comment by gabrieleborg** [comment]
Thanks @santiagohermo , such a nice exercise! 

I agree with the steps you have followed, and your main takeaways. In this sense, I am happy that we framed our results by reporting mostly the static, and the cumulative effect. They indeed are (more or less) the boundaries of the true effect depending on whether the strict exogeneity assumption works. 

I am wondering now how we can convey/integrate this piece of information in an easy way in the draft, because I believe that if we al agree that this is the case, it would really boost the strength of our results, as we are providing such boundaries.

**Comment by gabrieleborg** [comment]
Updated "Discussion" section: [see tasks](https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-742820148)

**Comment by diegogentilepassaro** [comment]
Great work @santiagohermo! We now know for a fact that strict exogeneity is enough for identifying the distributed lag model, so we can go ahead and claim it. The version of the draft that I committed had it written like that so there is nothing to do. We can just celebrate.

**Comment by diegogentilepassaro** [comment]
I am a little confused about the discussion. @santiagohermo and @gabrieleborg do you want to do a quick call to discuss this? 

I thought that we wanted to know whether strict exogeneity is enough to estimate correctly the distributed lag model. The answer is either yes or no. Our results are consistent with many things, but that has nothing to do with the answer to the following question, which I thought we were trying to answer:  
- If you assume strict exogeneity of the unobserved shocks to rents,  can we interpret our effect causally (sure you also need no anticipatory effects and a bit more)? 

My read of table 3 is: strict exogeneity is violated there and therefore you get the estimation of the true effects wrong. More generally, violations of strict exogeneity will yield almost surely the wrong estimates unless you cook into the estimation the structure of the errors, which we of course don't know in the real data.

**Comment by gabrieleborg** [comment]
Thanks for the comments @diegogentilepassaro 

I agree that we want to answer the question: "If you assume strict exogeneity of the unobserved shocks to rents, can we interpret our effect causally (sure you also need no anticipatory effects and a bit more)?"

But doesn't the exercise reported in columns 4, 5 and 6 of Table 2 confirm that? Meaning, when there is strict exogeneity of the shock, then conditional on all the assumptions we want to control for, the distributed lag model do actually picks up the true causal effect. Isn't this what we see in column 6, Table 2? despite using an additional lag, we get the (approximately) right estimates for the true effect which is t and t +1. 

In this sense, what I meant with boundaries is that, if instead there's no strict exogneity, Table 3 however show that the distributed lag model does a good job in estimating the static effect. 

I am fine if you guys want to talk :D

**Comment by santiagohermo** [comment]
Thanks @diegogentilepassaro! Let's iterate a bit more and if we still disagree we can jump on a call :)

> I thought that we wanted to know whether strict exogeneity is enough to estimate correctly the distributed lag model. The answer is either yes or no. Our results are consistent with many things, but that has nothing to do with the answer to the following question, which I thought we were trying to answer:
> 
> * If you assume strict exogeneity of the unobserved shocks to rents,  can we interpret our effect causally (sure you also need no anticipatory effects and a bit more)?

So yeah, the answer to that question is **yes**. Let me correct [my previous comment](https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-743353210). Strict exogeneity is sufficient for such interpretation, but is not necessary. Some people may disagree if we strongly say after the dynamic model that strict exogeneity is enough.

> My read of table 3 is: strict exogeneity is violated there and therefore you get the estimation of the true effects wrong. More generally, violations of strict exogeneity will yield almost surely the wrong estimates unless you cook into the estimation the structure of the errors, which we of course don't know in the real data.

Well, can it be the case that we are getting our estimate wrong then? We observe a two-period effect, can it be that the second period effect arises from an effect of the MW on the next-period unobservable?

---

More generally, I think that this may be thought of a matter of definition. Lagged MW either has an indirect effect through unobservables, or a direct effect. In the former case strict exogeneity doesn't hold, in the latter it does. Further, it seems that you can't possibly distinguish the two cases in the data, since you will be see an effect of lagged MW in your estimation either way.

As a result, I think we should focus our identification discussion in the static model, since this seems to be the substantive part.

**Comment by santiagohermo** [comment]
It seems we replied at the same time @gabrieleborg! Haha. If you guys want to jump to a (quick) call I'm in.

**Comment by santiagohermo** [comment]
I just saw your comment @gabrieleborg! I agree, we always seem to pick up the static effect correctly. That's really cool! If you guys want to jump to a quick call I'm in.

**Comment by diegogentilepassaro** [comment]
Let's do it @gabrieleborg and @santiagohermo! https://brown.zoom.us/my/dgentilepassaro

**Comment by santiagohermo** [comment]
Great discussion today @diegogentilepassaro @gabrieleborg. I learned a lot about panel models with this project :)

A few updates:

- In https://github.com/diegogentilepassaro/min_wage_rent/commit/3c3dbe32ae3f216097673d065ceeea67efc41a51 I updated the summary statistics tables dropping January 2010. I think everything looks good

- In https://github.com/diegogentilepassaro/min_wage_rent/commit/717d39f6f8a03da1ef1143457d807fcbdcfc1bf9 I ran the main model dropping the control **average wage** from the **professional and business sector**. Results are nearly identical. Should we keep this, saying we drop it because there seems to be an effect, but results don't change? Let me know what you think. If yes, I'll update the other folders as well (expmw, wgt, etc). If not we can just revert that commit.

**Comment by gabrieleborg** [comment]
@santiagohermo thanks!!

great for the table. As for the control, I would say let's vote, because I am fine either ways :)
I say we keep it. As Diego said in the meeting, we can also use it and explain why, given the lack of impact, we can consider it as a good way to then control for the economic cycle. 

I have updated the appendix explanation. Once we agree here I'll finish that up by adding both the rationale for keeping/not keeping that wage variable, and a description of the plots (plot labels updated here b4d4e74be7c70a6c78b87b7a9b67a7dbc198e793 in #75)

**Comment by santiagohermo** [comment]
I completed the appendix about the controls @diegogentilepassaro @gabrieleborg! In the end I went back to the previous version of the labels. The reason is that the middle coefficient also represents three months, which I had not taken into account before.

I'm not sure still what is the relationship between the monthly model coefficients and the quarterly model one. E.g., note that the monthly model coefficient uses 19 months (9 pre, month of change, 9 pots), whereas the quarterly model uses 21 months (9mo/3q pre, 3mo/1q of change, 9mo/3q post). It's not super important, but at some point we should really nail that down.

**Comment by diegogentilepassaro** [comment]
@gabrieleborg and @santiagohermo just posted my comments for discussion! Will move to conclusion :D

**Comment by santiagohermo** [comment]
For ease of reference, comments are tracked in https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-740755114

**Comment by gabrieleborg** [comment]
@diegogentilepassaro @santiagohermo 
guys I reviewed Diego's comments on discussion, there are some points that we still need to decide what to do :) 
Besides that thanks so much Diego!

**Comment by santiagohermo** [comment]
Completed review of first part of empirical strategy section. Annotated PDF updated [here](https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-740755114)

**Comment by gabrieleborg** [comment]
Just checked your revision @santiagohermo. For me everything's fine in that section.

**Comment by diegogentilepassaro** [comment]
@gabrieleborg just went through your comments, implemented the missing items in discussion, and updated the progress table [here](#65 (comment))

> @diegogentilepassaro @santiagohermo
> guys I reviewed Diego's comments on discussion, there are some points that we still need to decide what to do :)
> Besides that thanks so much Diego! updates [here](https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-740755114)

**Comment by diegogentilepassaro** [comment]
@santiagohermo I just took a look at your comments and changes of the comment below and they look good! 
> Completed review of first part of empirical strategy section. Annotated PDF updated [here](https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-740755114)

It seems that there is nothing else to discuss there 🥳

**Comment by santiagohermo** [comment]
Great @gabrieleborg @diegogentilepassaro! I'll mark the "finished" sections in our progress table.

**Comment by diegogentilepassaro** [comment]
My comments for conclusion now live in the progress table!

**Comment by santiagohermo** [comment]
I completed a review of "Emp Strategy II and Results I". Annotated pdf updated to [progress table](https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-740755114)

**Comment by santiagohermo** [comment]
I completed a review of the main results. Annotated pdf updated to [progress table](https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-740755114)

**Comment by santiagohermo** [comment]
> I completed a review of the experienced MW. Annotated pdf updated to [progress table](https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-740755114)

**Comment by santiagohermo** [comment]
I completed a review of the discussion section. Amazing job @gabrieleborg @diegogentilepassaro! It looks really cool. Annotated pdf updated to [progress table](https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-740755114)

**Comment by gabrieleborg** [comment]
Reviewed conclusion and addressed a couple of issues @diegogentilepassaro brought up! Annotated pdf in the [progress table](https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-740755114).

**Comment by santiagohermo** [comment]
I completed a review of abstract and introduction. Annotated pdf updated to [progress table](https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-740755114)

**Comment by gabrieleborg** [comment]
> I completed a review of "Emp Strategy II and Results I". Annotated pdf updated to [progress table](https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-740755114)

@santiagohermo thanks for the review. I am posting here since the Annotated PDF is now a rather old version and doesn't keep track of the latest change anymore. I am ok with most of the changes, however I was wondering whether the second paragrpah of 3.4 (description of exp MW empirical strategy, part on negative income shock) is a bit too technical for that part of the paper. since this is not our focus, I feel the reader may end up confused by what we are trying to achieve here. What does @diegogentilepassaro think about that? A possible solution could be to move this paragraph at bottom of section 5.1, where we introoduce the exp MW measure, show maps on how it differs from statutory etc.?

**Comment by gabrieleborg** [comment]
@santiagohermo @diegogentilepassaro I reviewed @santiagohermo' s comments and updates. I am ok with alomost all of them, so I didn't upload a further annotated PDF for all of the sections. Here a recap of my reviesion, and the last outstanding issues we may want to tackle today: 

- Intro and abtract: OK 
- Data: OK
- Emp. strategy: address Arellano Bond Formula + paragraph 3.4 location (see [my previous comment](https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-744488818))
- Results Pt. 1: OK (moved heterogeneity table to make it live in the right section. We may still want to play with this again)
- Results Pt.2: OK 
- Exp.MW: OK (see comment in emp. strategy)
- Discussion: OK (see minor comments in [Annotated pdf in progress table](https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-740755114))

After this, I give my green light!

**Comment by santiagohermo** [comment]
> > I completed a review of "Emp Strategy II and Results I". Annotated pdf updated to [progress table](https://github.com/diegogentilepassaro/min_wage_rent/issues/65#issuecomment-740755114)
> 
> @santiagohermo thanks for the review. I am posting here since the Annotated PDF is now a rather old version and doesn't keep track of the latest change anymore. I am ok with most of the changes, however I was wondering whether the second paragrpah of 3.4 (description of exp MW empirical strategy, part on negative income shock) is a bit too technical for that part of the paper. since this is not our focus, I feel the reader may end up confused by what we are trying to achieve here. What does @diegogentilepassaro think about that? A possible solution could be to move this paragraph at bottom of section 5.1, where we introoduce the exp MW measure, show maps on how it differs from statutory etc.?

I think that we do say something like that paragraph in section 5.1. So yeah, I'm okay with shortening that bit of section 3.4 (although it may be nice to have a brief thing mentioning why using both measures can be valuable)

Just make any change you want here. I'll later make a full read in order to spot any inconsistency that may arise after these last minute changes

**Comment by gabrieleborg** [comment]
Thanks @santiagohermo. I re-read it and in the end I think it works like this as well, so I won't touch it :)

**Comment by santiagohermo** [comment]
PDF tracking fixes of typos: [Main_SH.pdf](https://github.com/diegogentilepassaro/min_wage_rent/files/5693015/Main_SH.pdf) 
I also reviewed a few sentences in the Conclusion and splitted the write-up in several paragraphs

FYI @diegogentilepassaro @gabrieleborg

**Comment by santiagohermo** [comment]
In this branch we implemented a long run of production of the paper. The resulting PDF is [here](https://github.com/diegogentilepassaro/min_wage_rent/blob/a5d956bdcdceca9368eee69c579fe33d44c487f5/paper_slides/paper/Main.pdf). We've sent this version as dissertation and 3yp, and we expect to get a new round of feedback.

Changes merged to master in [this](https://github.com/diegogentilepassaro/min_wage_rent/commit/be45496a162423decb5033236eabb8cff955c64f) massive commit

### Main changes

- Review of figures and tables to make consistent throughout paper and write notes
- Re-writing of the "data" section. We now focus on talking about our Zillow and MW data, and we show the results of our balancing procedure more clearly. Importantly, we changed the descriptive tables quite a bit and we added the Zillow ZIP codes map
- Improvements in the "empirical strategy" section: (i) clearer discussion of the assumptions required for identification and pre-trends test, (ii) added rationale for selection economic controls and sent parametric trends to appendix, (iii) brief discussion on issues relating to experienced MW
- On the "rationale for selection economic controls" we have now an appendix estimating our model on QCEW variables.
- We improved the presentation of results with a tidier Table 3 and Figure 2. We improved this section in general with better presentation of results and (hopefully!) cleaner discussion
- We added a whole section on the experienced MW
- We added a discussion section with some benchmarking of the effects and policy implications
- Importantly, we realized that our geographic units are called "ZIP codes"!

### Some pending additions

- Add Appendix estimating model by aggregating our data to compare with literature
- Add "unbalanced reweighted" results to Table 4. We tried this and the results are very similar to the unbalanced panel (column 3). At the moment we are unsure why, so we didn't include them in the paper.
- We did some exploration and found that ZIP codes tend to enter in January, and less frequently in July (#73(. We ran models on a fully balanced panel from 2015 to 2019 and the results are similar (not included). Maybe we can do more here.
- Think about identification using experienced MW. (In particular, understand better residualized experienced MW. Maps?)
